CHANGELOG.md

```diff
@@ -35,10 +45,36 @@ Branch: `nilrt/master/next`
 - [Deprecated](https://github.com/ni/meta-nilrt/pull/663) the `ntpdate` package, since it has been dropped by upstream.
 - [Removed](https://github.com/ni/meta-nilrt/pull/673/commits/01ac041e848b67553001cee00336637eefb9384a) NIcurl-specific functions from curl, since these sources are no longer used by NIcurl.
 - [Removed](https://github.com/ni/meta-nilrt/pull/675) the unused `efifix` scripts.
+- [Removed](https://github.com/ni/meta-nilrt/pull/744) quagga entry from files group.
+- [Removed](https://github.com/ni/meta-nilrt/pull/749) sqlite3.
```

Comment:
We only removed sqlite3 from extras feed as it's already in main feed. From customer's perspective, this isn't important. Remove

Reply:
Fixed.